pylint_odoo/checkers/no_modules.py

```diff
@@ -378,11 +378,16 @@ def visit_call(self, node):
         if node.as_string().lower().startswith('fields.'):
             args = misc.join_node_args_kwargs(node)
             index = 0
+            field_name = ''
+            if (isinstance(node, astroid.Call) and
```

Comment:
This method is visit_call then you dont need a validation to check if this node is a call instance